packages/twenty-front/src/modules/ui/layout/show-page/components/ShowPageRightContainer.tsx

```diff
@@ -104,6 +105,7 @@ export const ShowPageRightContainer = ({
       {activeTabId === 'tasks' && <EntityTasks entity={entity} />}
       {activeTabId === 'notes' && <Notes entity={entity} />}
       {activeTabId === 'files' && <Attachments targetableEntity={entity} />}
+      {activeTabId === 'emails' && <Emails />}
```

Comment:
Can we hide this behind the messaging featureFlag?

Reply:
Done! The "Emails" tab will be disabled if `isMessagingEnabled` is false.